debian/rules

```diff
@@ -50,6 +50,7 @@ override_dh_auto_install:
 	install -D -m 644 udev/95-ceph-osd.rules $(DESTDIR)/lib/udev/rules.d/95-ceph-osd.rules
 	install -D -m 644 udev/60-ceph-by-parttypeuuid.rules $(DESTDIR)/lib/udev/rules.d/60-ceph-by-parttypeuuid.rules
 	install -D -m 644 src/etc-rbdmap $(DESTDIR)/etc/ceph/rbdmap
+	install -D -m 644 src/90-ceph-osd.conf $(DESTDIR)/etc/sysctl.d/30-ceph-osd.conf
```

Comment:
why the difference "90-ceph-osd.conf" -> "30-ceph-osd.conf" ?

Reply:
Here  I follow the /etc/sysctl.d/README  of ubuntu : "In general, files in the 10-*.conf range come from the procps package and serve as system defaults.  Other packages install their files in the
30-*.conf range, to override system defaults".
But in Centos I should use 90-*，so change 90-* to 30-* here.

Reply:
ACK